Wikilog.php

```diff
@@ -383,7 +383,7 @@ static function BeforePageDisplay( $output, $skin ) {
 	static function LinkBegin( $skin, $target, $text, $attribs, $query, &$options, &$ret )
 	{
 		if ( $target->isTalkPage() &&
-			( $i = array_search( 'broken', $options ) ) !== false ) {
+			( $i = array_search( 'broken', array_keys($options) ) ) !== false ) {
```

Comment:
Use `isset( $options['broken'] )` instead of array_search(array_keys())

Reply:
As I see, `$i` is needed for an offset later, in `array_splice`